kong/pdk/plugin.lua

```diff
@@ -6,6 +6,15 @@
 local _plugin = {}
 
 
+---
+-- Returns the instance ID of the running plugin.
+--
+-- @function kong.plugin.get_id
+-- @phases init_worker, rewrite, access, header_filter, response, body_filter, log
```

Comment:
does init_worker has a sense of plugin instance 🤔 ?

Reply:
It can work at this phase sometimes. But let's remove it.